docs/mutations.md

```diff
@@ -269,13 +348,127 @@ print(f"Before: {mts.num_mutations} mutations.\n"
+mts = msprime.sim_mutations(ts, rate=rate, model=jc69, random_seed=7)
+count_transitions(mts, jc69.alleles)
+```
+
+Although this model does not include silent mutations,
+applying it more than once could produce some -
+for instance, if one mutation with derived state ``C`` is inserted
+above an existing mutation with the same derived state.
+To make it state-independent, we have to choose the derived state
+uniformly from *all* nucleotides:
+
+```{code-cell}
+pi_jc69 = msprime.JC69MutationModel(state_independent=True)
+pi_jc69.transition_matrix
+```
+
+With this change, mutating twice with {class}`.JC69MutationModel`
+is *equivalent* to mutating once with rate equal to the sum of the rates.
+In other words, these two operations are equivalent:
+
+```{code-cell}
+alpha = 0.4 * rate
+beta = 0.6 * rate
+# mutate once:
+mts1 = msprime.sim_mutations(ts, rate=alpha + beta, model=pi_jc69, random_seed=7)
+print("Mutated once:")
+count_transitions(mts1, pi_jc69.alleles)
+# or, mutate twice:
+mts2 = msprime.sim_mutations(ts, rate=alpha, model=pi_jc69, random_seed=7)
+mts2 = msprime.sim_mutations(mts2, rate=beta, model=pi_jc69, random_seed=8)
+print("Mutated twice:")
+count_transitions(mts2, pi_jc69.alleles)
+```
+
+These are not identical, but similar.
+However, now 25% of the mutations are silent.
+These silent mutations are harmless, and will not affect most things at all.
+However, they do count as mutations, and so including them decreases the
+effective mutation rate.
+Above, we produced ``mts`` by mutating with the default Jukes-Cantor model
+at rate 1.5. Here, we produced ``mts1`` (and ``mts2``) using Jukes-Cantor
+but with 25% of the mutations silent.
+This means that to produce an set of mutations equivalent to ``mts``
+that includes silent mutations, we need to use the parent-independent
+Jukes-Cantor model with rate :math:`1/(1-0.25) = 4/3` times higher:
+
+```{code-cell}
+mts_with_silent = msprime.sim_mutations(ts, rate=rate * 4/3, model=pi_jc69, random_seed=12)
+count_transitions(mts_with_silent, pi_jc69.alleles)
+```
+
+Note that the offdiagonal counts are similar to what we saw above for ``mts``.
+Another way to see this is that nucleotide diversity is similar for ``mts`` and ``mts_with_silent``,
+but is only 75% as big for ``mts1``:
+
+```{code-cell}
+print(f"Nucleotide diversity for default JC69 with rate {rate}: {mts.diversity():.2f}")
+print(f"              parent-independent JC69 with rate {rate}: {mts1.diversity():.2f}")
+print(f"              parent-independent JC69 with rate {rate * 4/3}: {mts_with_silent.diversity():.2f}")
+```
+
+```{important}
+1. If you're going to add mutations to the same tree sequence more than once,
+    you should use the ``parent_independent`` mutation models
+    (or carefully consider the consequences).
+2. If you use a parent-independent model, you need to make your mutation rate higher,
+    to account for the silent mutations. See :ref:`sec_mutations_adjusting_for_silent`
```

Comment:
{ref} rather than :ref: (I'm considering adding some rules in my vimrc to do this automatically when I'm in a markdown file!)

Reply:
ah right. my least favorite thing about jupyterbook is how many different syntaxes there are.